tests/providers/test_foreman.py

```diff
@@ -139,3 +140,26 @@ def test_negative_remote_execution(foreman_stub):
 
     assert simple_result.status == 1
     assert complex_result.status == 100
+
+
+@pytest.mark.parametrize('response,expected,context',
+                         [
+                             ({1: 2}, {1: 2}, nullcontext()),
+                             ({"error": {"message": "Unable to authenticate user"}}, None, pytest.raises(AuthenticationError)),
+                             ({"error": {"full_messages": "foo", "message": "bar"}}, None, pytest.raises(ForemanBindError)),
+                             ({"errors": {"base": "foo"}}, None, pytest.raises(ForemanBindError)),
+                             ({"error": {"full_messages": ["bar"]}}, None, pytest.raises(ForemanBindError)),
+
+                         ]
+                         )
```

Comment:
here's some more ruff-style formatting. perhaps we should turn on formatting checks for the test directory.
```suggestion
@pytest.mark.parametrize(
    "response,expected,context",
    [
        ({1: 2}, {1: 2}, nullcontext()),
        (
            {"error": {"message": "Unable to authenticate user"}},
            None,
            pytest.raises(AuthenticationError),
        ),
        (
            {"error": {"full_messages": "foo", "message": "bar"}},
            None,
            pytest.raises(ForemanBindError),
        ),
        ({"errors": {"base": "foo"}}, None, pytest.raises(ForemanBindError)),
        ({"error": {"full_messages": ["bar"]}}, None, pytest.raises(ForemanBindError)),
    ],
)
```

Reply:
Yes, please, I prefer it very much when such problems' are solved by a machine.